src/System.Net.Http/src/System/Net/Http/Headers/KnownHeader.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+
+namespace System.Net.Http.Headers
+{
+    internal sealed class KnownHeader
+    {
+        private string _name;
+        private HttpHeaderType _headerType;
+        private HttpHeaderParser _parser;
```

Comment:
(you could also remove these explicit fields, make the properties { get; }, and then assign directly to those in the ctor... they'll be implicitly backed by readonly fields)